docs/deploy-cephfs.md

```diff
@@ -70,7 +70,7 @@ you're running it inside a k8s cluster and find the config itself).
 | `pool`                                                                                              | no             | Ceph pool into which volume data shall be stored                                                                                                                                                                        |
 | `volumeNamePrefix`                                                                                  | no             | Prefix to use for naming subvolumes (defaults to `csi-vol-`).                                                                                                                                                           |
 | `snapshotNamePrefix`                                                                                | no             | Prefix to use for naming snapshots (defaults to `csi-snap-`)                                                                                                                                                            |
-| `backingSnapshot`                                                                                   | no             | Boolean value. The PVC shall be backed by the CephFS snapshot specified in its data source. `pool` parameter must not be specified. (defaults to `false`)                                                               |
+| `backingSnapshot`                                                                                   | no             | Boolean value. The PVC shall be backed by the CephFS snapshot specified in its data source. `pool` parameter must not be specified. (defaults to `true`)                                                               |
```

Comment:
@riya-singhal31 Here you say the default is `true`. However, I looked a bit at the code and it seems there are 3 options:

- `backingSnapshot` is `true`, then the volume is backed by a snapshot or, if that's not possible, an error is returned.
- `backingSnapshot` is `false`, then the volume is never backed by a snapshot.
- `backingSnapshot` is not set, then the volume is backed by a snapshot if supported (i.e., `IsShallowVolumeSupported(req)` returns true.

Is my interpretation of the source code correct? If yes, then you cannot say that the default is `true`, because the default behavior deviates from the behavior in the `true`/`false` cases.